src/main/python/envbuilder/cli.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (c) 2020 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import logging
+import os
+import re
+
+from colors import color
+from progress.bar import ChargingBar
+from envbuilder import API
+from envbuilder import DockerImageSearch
+
+logger = logging.getLogger(__name__)
+
+class EnvBuilder():
```

Comment:
Do you need that ` ()` ?

Reply:
Hmm. Not sure. I will try that. Python is very new to me.